recipes/libnettle/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "nettle" %}
+{% set version = "3.3" %}
+{% set sha256 = "46942627d5d0ca11720fec18d81fc38f7ef837ea4197c1f630e71ce0d470b11e" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - configure.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]  
+
+requirements:
+  build:
+    - m4
+    - gcc
```

Comment:
This should be dropped as the VM contains a compiler already.

Reply:
When I only use gmp in the build requirements here, somehow gmp is not detected by configure and does not build hogweed in addition to nettle. However when I include gcc it is found and hogweed can be compiled. I need hogweed to build gnutls next so that's why I included gcc, do you have a suggestion on how to fix this ? Do you think I should mess again with the configure to make sure that gmp is detected when added here ?